grype/version/generic_constraint.go

```diff
@@ -8,9 +8,9 @@ import (
 var _ Constraint = (*genericConstraint)(nil)
 
 type genericConstraint struct {
-	raw        string
-	expression simpleRangeExpression
-	format     Format
+	Raw        string
+	Expression simpleRangeExpression
+	Fmt        Format
```

Comment:
nit: do these values need to be exported since a user can just use the interface methods? and then `fmt` could remain `format`?